### ISSUE TITLE
Supported Browsers

### ISSUE THREAD

**Issue by OmgImAlexis** [pymedusa/SickRage#1207]
We really need to get this sorted once and for all. I propose we drop support for all out of date browsers. If we only support the last 2 versions of Chrome, Firefox, Edge and Safari it means we no longer need anything to support old/broken browsers like IE.

This will also mean decreased page load times as we can take advantage of 99% of ES5 features and most of ES6 like `let`, template literals and a bunch of other fancy things that we currently can't use as they're not supported at all with IE or older versions of Chrome, etc.

PING: @pymedusa/administrators @pymedusa/contributors @pymedusa/developers @pymedusa/moderators @pymedusa/owners

**Comment by p0psicles** [comment]
I agree we should ditch IE8,9,10. But I do want to take small steps. I think it's a good idea to look into a browser support matrix for an Angular version? Or at least be practical in our choice. IMO we should first aim for a solid front end framework, before we start thinking about ES5 or ES6.

**Comment by OmgImAlexis** [comment]
Well either way all new frameworks support ES5 as a base and some even support a subset of ES6 features mainly the ones that can be poly filled. If you now we drop all of IE we can move to 99% of ES5 and around 40% of ES6 features.

**Comment by OmgImAlexis** [comment]
For now we're dropping all support for IE as well as Android Browser. The reason for IE is that it's superseded by Edge and all older OSs that don't support Edge can use Firefox and Chrome as there's a very limited amount of users still using IE with this. 

The reason for Android Browser is that it's been superseded by Android's Chrome which is a LOT closer to the desktop counterpart.

**Comment by OmgImAlexis** [comment]
The main features we gain with this is the CSS hacks for IE can be removed along with the polyfils and we can now use [template literals](https://kangax.github.io/compat-table/es6/#test-template_strings) and let in js.

Template literals can make code go from this.
var row = '';
        row += ' <tr class="' + $('#row_class').val() + ' show-' + indexerId + '">';
        row += '  <td class="tableleft" align="center"><input type="checkbox" class="' + indexerId + '-epcheck" name="' + indexerId + '-' + season + 'x' + episode + '"' + (checked ? ' checked' : '') + '></td>';
        row += '  <td>' + season + 'x' + episode + '</td>';
        row += '  <td class="tableright" style="width: 100%">' + name + '</td>';
        row += ' </tr>';

To this.
```
 return `
            <tr class="${$('#row_class').val()} show-${indexerId}">
                <td class="tableleft" align="center">
                    <input type="checkbox" class="${indexerId}-epcheck" name="${indexerId}-${season}x${episode}" ${(checked ? ' checked' : '')}>
                </td>
                <td>${season}x${episode}</td>
                <td class="tableright" style="width: 100%">${name}</td>
            </tr>
        `;
```
